renpy/display/swdraw.py

```diff
@@ -306,12 +306,13 @@ def draw_special(what, dest, x, y):
 
         ramplen = what.operation_parameter
 
-        ramp = b"\x00" * 256
+        from itertools import repeat
 
-        for i in range(0, ramplen):
-            ramp += bchr(255 * i // ramplen)
-
-        ramp += b"\xff" * 256
+        ramp = bytes([
+            *repeat(0, 256),
+            *(255 * i // ramplen for i in range(ramplen)),
+            *repeat(255, 256),
+        ])
```

Comment:
Aside from needing a separate import, this seems actively harder to read now. Perhaps instead only replace the `for` loop with `ramp += bytes(255 * i // ramplen for i in range(ramplen))`?